### PR TITLE
Backend/database/setup

### PR DESCRIPTION
**Pull request by Nogeenaccount**
Database is now getting populated

### CODE REVIEW

File: backEnd/data/edges.csv [hunk 1]
```diff
@@ -0,0 +1,34 @@
+start,end,genome
```
Comment (MatthijsRijlaarsdam) [inline backEnd/data/edges.csv]: What do you mean by genome? Why is your testfile this large?

File: backEnd/data/nodes.csv [hunk 2]
```diff
@@ -0,0 +1,16 @@
+id,sequence
```
Comment (MatthijsRijlaarsdam) [inline backEnd/data/nodes.csv]: What kind of nodes are these?

File: backEnd/src/test/java/database/DatabaseTest.java [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+    }
+
+    /**
+     * Test if all nodes from the test file get inserted.
+     */
+    @Test
+    public void testEdgeInsertion() {
+        Iterable<Relationship> allRelas;
+        Set<String> strands = new HashSet<>();
+        strands.add("1|2 AA");
```
Comment (MatthijsRijlaarsdam) [inline backEnd/src/test/java/database/DatabaseTest.java]: this method is too long.

File: backEnd/src/test/java/database/DatabaseTest.java [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+import org.neo4j.graphdb.Relationship;
+import org.neo4j.graphdb.Transaction;
+import org.neo4j.tooling.GlobalGraphOperations;
+import parser.Parser;
+
+import java.io.File;
+import java.util.HashSet;
+import java.util.Set;
+
+/**
```
Comment (MatthijsRijlaarsdam) [inline backEnd/src/test/java/database/DatabaseTest.java]: missing a comment

**Comment by MatthijsRijlaarsdam** [comment]
Looks good, but testing is not up to par as there are far too large test files (where you can make errors too). Try to find a different way to test these files. Also, this is not connected to anything yet, make sure we actually use this. Finally, you only load in strains, but we have a datatree as well, try to load that in to the database as well.

**Comment by jeffreyhelgers** [comment]
In the database class there are a few unused variables and imports.

File: backEnd/src/main/java/controller/Controller.java [hunk 5]
```diff
@@ -322,11 +326,63 @@ public String loadRawFileData(String fileName) {
 				rawFileData = rawFileData + line;
```
Comment (jeffreyhelgers) [inline backEnd/src/main/java/controller/Controller.java]: Findbugs gives an error here.

**Pull request closed without merging**